LayoutTests/webanimations/offset-anchor-animation-yields-compositing.html

```diff
@@ -0,0 +1,15 @@
+<pre id="results"></pre>
+<div id="target" style="width: 100px; height: 100px; background-color: black; offset-path: path('M10,10 H50')"></div>
```

Comment:
why set the background color?

Reply:
So that something visually renders when running that test in the browser.